lib/json_info_codec.dart

```diff
@@ -7,10 +7,11 @@ part of dart2js_info.info;
 
 // TODO(sigmund): add unit tests.
 class JsonToAllInfoConverter extends Converter<Map<String, dynamic>, AllInfo> {
-  Map<String, Info> registry = <String, Info>{};
+  final Map<String, Info> _registry =
+      new SplayTreeMap<String, Info>(compareNatural);
```

Comment:
Updated reasoning. It's good to use `SplatTreeMap` just because it's faster – when you don't care about stable output.